package.json

```diff
@@ -2,17 +2,19 @@
   "name": "webostv-as-promised",
   "version": "0.0.2",
   "description": "Wrapper library webOSTV.js on Promise",
-  "main": "dist/index.js",
+  "main": "./bundle.js",
+  "type": "index.d.ts",
```

Comment:
`types`

[`type` field will be used for es modules.](https://nodejs.org/api/esm.html#esm_package_json_main)

Reply:
?

Reply:
Ok